flexibee_backend/models/fields.py

```diff
@@ -163,12 +163,19 @@ def __unicode__(self):
         return '.'.join((self.instance.pk, self.field.type)) if self.instance else None
 
     @property
+    def language_code(self):
+        lang_code = translation.get_language()
+        if not lang_code in config.FLEXIBEE_PDF_REPORT_AVAILABLE_LANGUAGES:
+            lang_code = config.FLEXIBEE_PDF_REPORT_DEFAULT_LANGUAGE
+        return lang_code
+
+    @property
     def file_response(self):
         if not self.instance.pk:
             raise AttributeError('The object musth have set id.')
-
         connector = get_connector(ModelConnector, self.instance.flexibee_company.flexibee_db_name)
-        r = connector.get_response(self.instance._meta.db_table, self.instance.pk, self.field.type)
+        query_string = 'report-lang=%s' % self.language_code
```

Comment:
Tak nakonec potřebujeme rozlišit jen jazyk, typy faktur se zatím nebudou dělat :-)